docs/assets/js/main.js

```diff
@@ -9,7 +9,7 @@ import AlphaTransition from '@cfpb/cfpb-atomic-component/src/utilities/transitio
 import MoveTransition from '@cfpb/cfpb-atomic-component/src/utilities/transition/MoveTransition.js';
 import MaxHeightTransition from '@cfpb/cfpb-atomic-component/src/utilities/transition/MaxHeightTransition.js';
 import Table from '@cfpb/cfpb-tables/src/Table';
-import { Tabs } from 'govuk-frontend';
+import Tabs from './tabs'
```

Comment:
We're obviously not consistent with this, but sometimes I think it's a good idea to add the `.js` because then you can search for the file name and see where it's imported.